csrc/torch_bindings.cpp

```diff
@@ -133,6 +133,15 @@ TORCH_LIBRARY_EXPAND(TORCH_EXTENSION_NAME, ops) {
   ops.def("gptq_marlin_24_gemm", &gptq_marlin_24_gemm);
   ops.impl("gptq_marlin_24_gemm", torch::kCUDA, &gptq_marlin_24_gemm);
 
+  // Machete (Dense) Optimized Mixed Precision GEMM for Hopper.
+  ops.def("machete_supported_schedules", &machete::supported_schedules);
+  ops.impl("machete_supported_schedules", torch::kCPU,
+           &machete::supported_schedules);
+  ops.def("machete_gemm", &machete::gemm);
+  ops.impl("machete_gemm", torch::kCUDA, &machete::gemm);
+  ops.def("machete_prepack_B", &machete::prepack_B);
+  ops.impl("machete_prepack_B", torch::kCUDA, &machete::prepack_B);
```

Comment:
Use string schemas for these functions.  Otherwise the device specific impls (and any meta functions) are ignored.

Reply:
Done